### DISCUSSION TITLE
how disinformation feeds cognitive dissonance

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#150] [General]
Ethan Siegel's disinformation is also noticed by Stacy McGaugh.

Comment on "Triton Station", Stacy McGaugh's blog
> [Apass] September 14, 2023 at 2:56 pm
> And the sad part is that Ethan Siegel will cite only that paper [arXiv:2309.05252] and not also your answer and will not point that is also problematic for LCDM. And that will strengthen the public opinion about MoND = bad, LCDM = good.

> > [tritonstation] September 15, 2023 at 3:47 pm
> > Yep. That’s how disinformation feeds cognitive dissonance.

[https://tritonstation.com/2023/09/08/is-the-milky-ways-rotation-curve-declining/](https://tritonstation.com/2023/09/08/is-the-milky-ways-rotation-curve-declining/)

**Comment by bgreenman** [comment]
The reason MoND is not taken seriously by researchers is that most researchers have invested their energy and time on working with LCDM

**Reply by sahil5d** [reply]
Another, smaller reason is that the name is dissonant. They should rebrand as Milgrommatic gravity or M gravity imo.

**Reply by bgreenman** [reply]
Thats a great idea
